spec/mdn/mdn.mdown

```diff
@@ -0,0 +1,88 @@
+# Introduction
+
+JMAP <https://tools.ietf.org/html/draft-ietf-jmap-core-06> is a generic protocol for synchronising data, such as mail, calendars or contacts, between a client and a server. It is optimised for mobile and web environments, and aims to provide a consistent interface to different data types.
+
+This specification defines a method helping to send MDN (Message Disposition Nodification) messages. MDN are defined in [@!RFC8098] and are used as "read receipts", "acknowledgements", or "receipt notifications".
+
+## Notational conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!RFC2119].
+
+Type signatures, examples and property descriptions in this document follow the conventions established in Section 1.1 of <https://tools.ietf.org/html/draft-ietf-jmap-core-06>.
+Email object and methods are defined in <https://tools.ietf.org/html/draft-ietf-jmap-mail-06>.
+
+## Terminology
+
+The same terminology is used in this document as in the core JMAP specification.
+
+## Addition to the capabilities object
+
+The capabilities object is returned as part of the standard JMAP Session object; see the JMAP spec. Servers supporting *this* specification MUST add a property called `urn:ietf:params:jmap:mdn` to the capabilities object.
+
+
+# Email/createMDN
+
+The Email/createMDN method create a [@!RFC5322] message from MDN properties.
+
+It takes the following arguments:
+
+- **accountId**: `String|null`
+  The id of the account to use for this call. If `null`, defaults to the `urn:ietf:params:jmap:mail` primary account.
+- **mdns**: `String[MDN]`
+  A map of creation id (client specified) to MDN objects
+
+An **MDN** object has the following properties:
+
+- **creationId**: `String`
+  (client specified) Used to keep track of the MDN.
+- **referencedMessageId**: `String`
+  Message Id of the received message the user wants to send an MDN for.
+- **subject**: `String`
+  Subject that will be used as `Subject` header for this MDN.
+- **textBody**: `String`
+  Human readable part of the MDN, as plain text.
+- **reportingUA**: `String`
+  Name of the MUA creating this MDN. It is used to build the MDN Report part of the MDN.
+- **disposition**: `Disposition`
+  Object containing the diverse MDN disposition options.
+
+  Disposition is defined with the following fields:
```

Comment:
You need \`\` here like `Disposition` no? Or \*\*Disposition\*\*.

I'll even match what you wrote for MDN:

```
An **Disposition** object has the following properties:
```